tasks/managed/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       default: "3"
     - name: concurrentLimit
       type: string
-      default: 5
+      default: 90
```

Comment:
I think we've discussed this in a previous PR somewhere, but can't seem to find it. What was the outcome of that, did we decide on bumping the concurrent limit to this much?

Reply:
We talked about that here: https://github.com/konflux-ci/release-service-catalog/pull/1207#discussion_r2207324790

I have an AI to test these changes with a smaller Snapshot, but I don't know if there were any other blocking concerns about this.

@johnbieren @theflockers

Reply:
I tested this with a smaller snapshot and the overall performance of the pipeline was no worse than before.